src/lib/components/article-link.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+	import Launch from '$lib/components/icons/launch.svelte';
+
+	export let href: string | undefined = undefined;
+	export let target: string | undefined = '_blank';
```

Comment:
I don't think that links should default to `target="_blank"`.

Reply:
No worries - I figured the majority of the links would be to open the single article page from the accordion, and now I'm thinking that's probably not the case. But the _blank thing is always debatable/controversial 😅 and I have no strong feelings either way about that so let's change it 😉